.docker/Dockerfile.govcms7

```diff
@@ -1,9 +1,34 @@
-FROM govcmsdev/govcms7
+FROM amazeeio/php:7.1-cli-drupal as builder
+
+ENV GOVCMS_PROJECT_VERSION=7.x-3.x
```

Comment:
if we continually pull from the head of the dev branch - we'll never actually get a proper point-in-time tagged release (making rollbacks, reverts and re-creations hard) - this project version should increment with the release of GovCMS that this is tied to.

Reply:
@tobybellwood there was no tagged release of the `7.x-3.x` branch, but it can be updated to build from a tagged release. Will also need a change to the makefile to set version rather than branch.

Reply:
Further to this we could skip the make process completely if the version was published on D.O and we can fetch the codebase version directly from there.

Reply:
happy to leave it on GitHub - gives us more flexibility to build non-prod branches for testing.